template/src/ontology/Makefile.jinja2

```diff
@@ -707,17 +725,14 @@ mirror-{{ ont.id }}: | $(TMPDIR)
 {%- endif %}
 {%- endif %}
 {%- if ont.is_large %}
-else
-mirror-{{ ont.id }}:
-	@echo "Not refreshing {{ ont.id }} because refreshing large imports is disabled (IMP_LARGE=$(IMP_LARGE))."
```

Comment:
Some message at run.sh level indicating the IMP MIR COMP parameters might be useful when the job is finished but no sweat. This can help for example when custom workflows with parameter configs are defined (like prepare_release_fast with IMP = false), and you just want to tell the use: just so you know, you didn't refresh imports.

Reply:
Why not, but then displaying the message at the moment a target is skipped would not be very useful – the message would undoubtedly be lost in the middle of the entire Make output.

If we want such a message, better to do that explicitly at the end of a workflow – something like:

```Make
prepare_release: all_odk
      [ ... normal commands of the prepare_release target ... ]
      @echo "Release files are not in $(RELEASEDIR) - now you should commit, push and make a release on your Git hosting site."
      @if [ "$(IMP)" != true ]; then echo "Note: Imports were not refreshed." ; fi
```

Reply:
Yes that is probably better. I was thinking of something at odk wrapper level to always print this information but that is silly since many pipelines do not care about these three variables.. Thanks!